pages/graphql.vue

```diff
@@ -696,6 +738,20 @@ export default {
         this.$toast.info(this.$t("finished_in", { duration }), {
           icon: "done",
         })
+
+        entry = {
+          ...entry,
+          response: this.response,
+          date: new Date().toLocaleDateString(),
```

Comment:
```suggestion
          updatedOn: new Date(),
```
We generally use `updatedOn` attribute to store timestamps. Also, saving timestamp as string isn't preferred. Firebase support date datatype. Must leverage that.

Reply:
So I've tried to follow the format used in index.vue for storing REST requests. Should I change it there too?

Reply:
That would be good.